src/components/reusable/breadcrumbs/breadcrumbItem.scss

```diff
@@ -0,0 +1,23 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/foundation/scss/mixins/typography.scss';
+
+.breadcrumb-item {
+  @include typography.type-ui-01;
+
+  display: inline;
+  color: var(--kd-color-text-primary);
+  font-weight: var(--kd-font-weight-medium);
+
+  a {
+    text-decoration: none;
+    color: var(--kd-color-text-tertiary);
+    border-radius: 4px;
+
+    &:focus,
+    &:active {
+      padding: 16px;
```

Comment:
The entire component shifts on the page due to adding padding for focus state. We need to change it so that doesn't happen. Make it so it always has a transparent outline, then you can transition the outline-color.

Reply:
Well, since the gap between the separator and the left and right content is 16px. If there's only an outline with the offset of 16px, it'll touch the separator.

Reply:
Can take that up with Robert to see what he thinks. Maybe reduce the outline spacing to 8px?

Reply:
Sure, asked him for a design where everything is there so everyone can be on the same page.